oleesir/Banking-app
# #164723354Users should be able to see the sign up page

#### What does this PR do?
implement signup-template
#### Description of Task to be completed?
implement signup-template
#### How should this be manually tested?
* Navigate to the UI folder and click on signup.html in your browser
* You will see a signup-template
#### What are the relevant pivotal tracker stories?
[#164723354](https://www.pivotaltracker.com/story/show/164723354)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-20 at 10 59 38 PM" src="https://user-images.githubusercontent.com/17679511/54724926-68022400-4b6d-11e9-97b1-65030569e0da.png">
